### DISCUSSION TITLE
Workflow fails because of the release name?

### DISCUSSION THREAD

**Discussion by justalemon** [ncipollo/release-action#369] [General]
I'm having quite of a weird behavior here, and I ain't sure if is a bug or I missed something.

I have [this workflow file](https://github.com/justalemon/ActionsTesting/blob/master/.github/workflows/main.yml) that triggers when I create a release, and it causes some weird behavior depending on what I name the release.

- If I call it vN, for example, [v1](https://github.com/justalemon/ActionsTesting/releases/tag/v1), [it fails](https://github.com/justalemon/ActionsTesting/actions/runs/6205133106) with "No tag found in ref or input"
- If i call it N, for example, [2](https://github.com/justalemon/ActionsTesting/releases/tag/v2), [it finishes sucessfully](https://github.com/justalemon/ActionsTesting/actions/runs/6205209346)

Is that normal? I would expect it to take the name from the release payload data so it *should* have the tag.

**Comment by ncipollo** [comment]
You need to either manually set the tag or trigger your workflow from something like a tag push. This will try and use the ref GitHub associated with the workflow but I have no idea what that will be when release creation is the trigger. Further, trigging this action which creates a release from release creation seems like it could cause some weird issues generally.

Usually you would trigger you workflow from a tag push, which then creates a release, uploads assets, etc.

**Reply by ncipollo** [reply]
You can check out the logic for extracting the tag from the ref provided by GitHub here:

https://github.com/ncipollo/release-action/blob/main/src/Inputs.ts#180

Guessing that's something weird when the trigger is itself a release (again the intent is to trigger from something like a tag push if you want the action to set the tag for you).
